chapter-10-workload-identities/demoapi/README.md

```diff
@@ -66,6 +66,14 @@ spec:
             readOnly: true
 ```
 
```

Comment:
Added these notes for the reader.
Although all of these annotations need understanding, once learned it is straightforward to scale them.